### PR TITLE
Redefine transaction

### PR DESCRIPTION
**Pull request by flywire**
Describe what a transaction actually is. Informed by https://www.investopedia.com/terms/t/transaction.asp .

If `key-value frame` is required it should be written in simpler terms.

### CODE REVIEW

**Comment by fellen** [comment]
Hm, perhaps we should distinguish like [wikipedia](https://en.wikipedia.org/wiki/Transaction):
'''Transaction''' or '''transactional''' may refer to:
==Commerce==
*[[Financial transaction]], an agreement, communication, or movement carried out between a buyer and a seller to exchange an asset for payment
*Debits and credits in a [[Double-entry bookkeeping system]]
…
@jralls ?

**Comment by jralls** [comment]
This PR is misguided. The definition of transaction here is how it's used in GnuCash, which is appropriate for GnuCash's glossary. Users should consult a [dictionary](https://www.oed.com/search/dictionary/?scope=Entries&q=transaction) for the plain-language meaning or an [accounting text](https://www.investopedia.com/terms/t/transaction.asp) for the accounting meaning.
